src/components/RocketProfile.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const RocketProfile = () => {
+  const reserved = useSelector((store) => store.rockets.filter((rocket) => rocket.rocketStatus));
+  return (
+    <div className="profileRocket">
+      <h2>My Rockets</h2>
+      {reserved.map((rocket) => (
+        <span key={rocket.rocketId}>
+          {rocket.rocketName}
+        </span>
+      ))}
```

Comment:
[OPTIONAL] Well done overall 💯  but the My Profile section could be enhanced by adding a placeholder message when the "My Missions" or "My Rockets" lists are empty (no missions joined or no rockets reserved). It would go like this 👇🏽 

```
      {reserved.length > 0 ? (
        reserved.map((rocket) => (
          <span key={rocket.rocketId}>
            {rocket.rocketName}
          </span>
        ))
      ) : (
        <span>No reserved rockets</span>
      )}
```